src/Queries/Update.php

```diff
@@ -5,6 +5,9 @@
 
 use SimpleCrud\Table;
 
+/**
+ * @method self where(string $condition, ...$bindInline)
+ */
```

Comment:
There are more methods (the list in the constant `ALLOWED_METHODS`). Not sure if it's a good idea because require more maintenance work.